tree/ntupleutil/v7/inc/ROOT/RNTupleImporterCLI.hxx

```diff
@@ -0,0 +1,59 @@
+/// \file ROOT/RNTupleImporterCLI.hxx
+/// \ingroup NTuple ROOT7
+/// \author Florine de Geus <florine.de.geus@cern.ch>
+/// \date 2023-10-25
+/// \warning This is part of the ROOT 7 prototype! It will change without notice. It might trigger earthquakes. Feedback
+/// is welcome!
+
+/*************************************************************************
+ * Copyright (C) 1995-2023, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+/* This header contains helper functions for the ttree2rntuple program
+   for CLI related actions, such as argument parsing and output printing. */
+
+#ifndef ROOT7_RNTupleImporterCLI
+#define ROOT7_RNTupleImporterCLI
+
+#include "ROOT/RNTupleOptions.hxx"
+
+#include <vector>
+
+namespace ROOT {
+namespace Experimental {
+namespace RNTupleImporterCLI {
+
+struct ImporterConfig {
+   /// By default, compress RNTuple with zstd, level 5
+   static constexpr int kDefaultCompressionSettings = 505;
```

Comment:
Is this the same as the default for a new `RNTuple`?

Reply:
It is the same as the default for a new _imported_ `RNTuple`. However, the `RNTupleWriter` uses the `RNTupleWriteOptions` default which is`RCompressionSetting::EDefaults::kUseAnalysis` (which is 404). @jblomer is this intentional, or should we change this?